### PR TITLE
update travis file with black formatter

### PR DESCRIPTION
**Pull request by deepcompute-agent**

### CODE REVIEW

**Comment by deepcompute-agent** [comment]
* The `black` tool will install `>=python3.6` but the current version is `3.5` so that's why the travis check had failed. so someone has take responsibility for this.

File: .travis.yml [hunk 1]
```diff
@@ -1,40 +1,43 @@
...
-    - kwikapi/__init__.py
+  - README.md
+  - setup.py
+  - LICENSE
+  - kwikapi/api.py
+  - kwikapi/__init__.py
   name: kwikapi-0.5.9
   tag_name: 0.5.9
-  on:
+  true:
```
Comment (jaswanth098) [inline .travis.yml:16]: This should not be changed I guess @Rajendra-R

**Comment by jaswanth098** [comment]
@Rajendra-R Could you check this

File: .travis.yml (hunk 1 above)
Comment (ghost) [reply .travis.yml:16]: * We are using python `PyYAML` module it is doing the above conversion but we checked there is no problem with that.

**Comment by deepcompute-agent** [comment]
* To change the python version we have to convert the entire code, though we are not working on this repository we are closing this pull request.

**Pull request closed without merging**